third-party/mcrouter/mcrouter-CMakeLists.txt

```diff
@@ -98,5 +98,5 @@ target_include_directories(mcrouter PUBLIC "${CMAKE_CURRENT_BINARY_DIR}" "${JEMA
 
 install(
   TARGETS mcrouter
-  LIBRARY ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
+  LIBRARY ARCHIVE DESTINATION lib
```

Comment:
Why not use the variable here too?

Reply:
Updated

Reply:
Updated

Reply:
I think we should remove the `DESTINATION` because we are installing a library to the default destination.